packages/server/migrations/201907100048002-templates.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.createTable('Templates', {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        name: {
+          type: Sequelize.STRING(127),
+          allowNull: false,
+        },
+        envelopeTemplate: {
```

Comment:
I’m not familiar with the phrase “envelope” here. Is this basically an email template, or a general message template?

Reply:
The field "envelopeTemplate" is used to save templates for any systems needs, so, the better to rename it into something else here, for instance "template", but "template" is reserved term for model name, so to make difference just let's name it as "envelopeTemplate".